test/unit/finance/billing_strategy_test.rb

```diff
@@ -78,7 +78,7 @@ def setup
     canaries = FactoryBot.create_list(:provider_with_billing, 4).map(&:id)
     ThreeScale.config.payments.expects(:billing_canaries).at_least_once.returns(canaries)
 
-    Finance::BillingStrategy.expects(:daily_async).with { |scope| assert_equal canaries, scope.pluck(:account_id) }.returns(true)
+    Finance::BillingStrategy.expects(:daily_async).with { |scope| assert_same_elements canaries, scope.pluck(:account_id) }.returns(true)
```

Comment:
This probably doesn't belong to the PR, but I don't want to open a YET ANOTHER PR for fixing this randomly failing test... https://app.circleci.com/pipelines/github/3scale/porta/30455/workflows/3985f45a-8be0-47ef-8b67-5fe032259a50/jobs/341167/tests